### PR TITLE
OD-111 [Task] Grammatical number message correct spelling for delete forever action

### PR DESCRIPTION
**Pull request by ivandevupp**
@sofiiakvasnevska
## Issue
OD-111 https://weboo.atlassian.net/browse/OD-111
## Description
Grammatical number message correct spelling for delete forever action
## Screenshots/screencasts
https://monosnap.com/file/PGnQclpDohB5To3ikcWmmF9xlnDR3V
## Backward compatibility
This change is fully backward compatible.
## Reviewers 
@upplabs-alex-levchenko @AndrRyaz

### CODE REVIEW

File: js/interface.js [hunk 1]
```diff
@@ -1511,7 +1511,9 @@ $('.file-manager-wrapper')
     var $items = $('.file-row.active');
     Fliplet.Modal.confirm({
       titile: 'Delete items',
-      message: 'The action will delete the file forever and it can not be undone.',
+      message: 'The action will delete '
+        + ($items.length > 1 ? 'files' : 'the file')
```
Comment (tonytlwu) [inline js/interface.js:1515]:
```suggestion
        + ($items.length > 1 ? $items.length + ' files' : 'the file')
```

Comment (tonytlwu) [inline js/interface.js:1515]:
```suggestion
        + ($items.length > 1 ? $items.length + ' files' : 'the file')
```

**Review by tonytlwu — CHANGES_REQUESTED** [review]

File: js/interface.js (hunk 1 above)
Comment (ivandevupp) [reply js/interface.js]: Fixed.

Comment (ivandevupp) [reply js/interface.js]: Fixed.

**Review by tonytlwu — APPROVED** [review]
@sofiiakvasnevska This is ready for testing

Risk: Low
Affected: Studio users deleting files in Studio
Verify:  File deleting is working correctly

Squash and merge: Use PR title

**Pull request merged**